packages/cubejs-client-react/src/useCubeQuery.js

```diff
@@ -11,11 +11,21 @@ export default (query, options = {}) => {
   const [currentQuery, setCurrentQuery] = useState(null);
   const [isLoading, setLoading] = useState(false);
   const [resultSet, setResultSet] = useState(null);
+  const [progress, setProgress] = useState(null);
   const [error, setError] = useState(null);
   const context = useContext(CubeContext);
 
   let subscribeRequest = null;
 
+  const progressCallback = ({ progressResponse }) => {
+    setProgress({
```

Comment:
Let's just save the `ProgressResponse` directly and let the user customize it on their side

Reply:
I can do that, but then the code for consuming the progress response will be much more convoluted?

I.e. the caller needs to access `.stage.stage` to get the message 🤷‍♂️ 

```
 export default function App() {
   *   const { resultSet, isLoading, error } = useCubeQuery({
   *   const { resultSet, isLoading, error, progress } = useCubeQuery({
   *     measures: ['Orders.count'],
   *     dimensions: ['Orders.createdAt.month'],
   *   });
   *
   *   if (isLoading) {
   *     return <div>Loading...</div>;
   *     return <div>{progress && progress.stage && progress.stage.stage || 'Loading...'}</div>;
   *   }
```

Reply:
Updated now 👍